models/OrderHeader.js

```diff
@@ -0,0 +1,28 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('OrderHeader', {
+		order_user_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			references: {
+				model: 'User',
+				key: 'id'
+			}
+		},
+		order_supplier_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false
+		},
+		order_item_count: {
```

Comment:
What is the purpose of this field? This will have to be updated when an order item is added?

Reply:
Just total count of items in an order. Something like you see on a Costco/Walmart receipt.

Can be calculated as well. Let us decide on Saturday